staging/src/k8s.io/controller-manager/config/v1alpha1/conversion.go

```diff
@@ -18,8 +18,10 @@ package v1alpha1
 
```

Comment:
this file should be split up so that this repo only contains conversions for GenericControllerManagerConfiguration, and the rest of the conversions are moved to the cloud-provider repo